src/hatti/views/table.cljs

```diff
@@ -187,6 +199,72 @@
                event (aget dataview handler-name)]]
      (.subscribe event handler-function))))
 
+(defn add-element
+  [vector value]
+  (-> vector
+      (conj (int value))
+      set
+      vec))
+
+(defn remove-element
```

Comment:
Rather generic helper could potentially go in Chimera

Reply:
Cool, I'll move both `add-element` and `remove-element` to Chimera